dogowner/apps.py

```diff
@@ -0,0 +1,6 @@
+from django.apps import AppConfig
+
+
+class DogownerConfig(AppConfig):
```

Comment:
The class name should be camelcase, see https://realpython.com/python-pep8/#naming-styles